spec/01-unit/006-schema_validation_spec.lua

```diff
@@ -64,6 +64,45 @@ describe("Schemas", function()
       end)
     end)
 
+    describe("[string]", function()
+      it("should trim whitepsace from value with no trim_whitespace property set", function()
+        local values = {string = " kong "}
+
+        local valid, err = validate_entity(values, schema)
+        assert.True(valid)
+        assert.falsy(err)
+        assert.are.same("kong", values.string)
+      end)
+      it("should trim whitepsace from value with trim_whitespace = true", function()
+        local trimSchema = {
```

Comment:
a nitpick, but can we use snake_case instead for these? Thanks in advance :)

There are a few typos in the tests name that we can fix ourselves before merging too, no worries.